includes/TripalFields/local__interpro_results/local__interpro_results.inc

```diff
@@ -54,6 +54,9 @@ class local__interpro_results extends ChadoField {
 
     if (empty($interpro_results)) {
       unset($entity->{$field_name});
```

Comment:
```suggestion
```

Reply:
I think it might because there are some default values? I recall having the issue on tripal_analysis_go.
I'll test to make sure there are not stray values.

Reply:
I see your point. You might want to explicity set it rather than its children:

```php
$entity->{$field_name} = array(
  'und' => array(
    array('value' => '')
  )
);

Reply:
That way you don't need to unset then re-set.

Reply:
Sure, that would be fine. I'll add it